seedseeker/generators/ran3.py

```diff
@@ -0,0 +1,73 @@
+from typing import Iterator
+import random
+
+
+def ran3(seed: int):
+    """
+    based on the C# implementation
```

Comment:
add a link to it, the book or the issue, please (I should've done similarly for the others, will fix)